lib/app/app_startup.dart

```diff
@@ -11,6 +14,12 @@ Future<void> appStartup(AppStartupRef ref) async {
     ref.invalidate(sharedPreferencesProvider);
   });
   await ref.watch(sharedPreferencesProvider.future);
+  final flavor = getFlavor();
+
+  if (!kIsWeb && flavor == Flavor.prod) {
+    final notifications = ref.read(pushNotificationsProvider.notifier);
+    notifications.initializePushNotifications();
```

Comment:
let's call this notificationsManager or notificationsService or similiar and the class as well

Reply:
please :)